src/rgw/store/dbstore/common/dbstore_log.h

```diff
@@ -12,7 +12,6 @@
 #include <fstream>
 #include "common/dout.h"
 
-#define dout_subsys ceph_subsys_rgw
 #undef dout_prefix
 #define dout_prefix *_dout << "rgw dbstore: "
```

Comment:
Ideally we should probably not define `dout_prefix` in a header file, either, but I think the only way around that would be making a DoutPrefixProvider.